docs/glossary.md

```diff
@@ -47,6 +51,18 @@ In each block, five tickets are called to vote. In addition to votes on any open
 
 ---
 
+#### Coinbase maturity
+
+Outputs from coinbase and stakebase transactions cannot be spent until the coinbase maturity period has passed. The coinbase maturity period is 256 blocks.
+
+#### Coinbase transaction
+
+Every block which is mined contains a single coinbase transaction. This transaction will only have one input, and that is newly created Decred which constitutes the PoW and Treasury portions of the block reward. Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if proof-of-stake voters vote to reject the block which contains them. Decred created in coinbase transactions cannot be spend until the coinbase maturity period has passed.
```

Comment:
This seems to be a recurring them, but I really don't think the Glossary should be calling out specific details of how things are composed.  That is more the purpose of developer documentation, imo.

These things can and will change over time in Decred due to the ability for consensus rule voting.  It is almost positively certain this glossary will become outdated and incorrect if it includes specific details.

The glossary should, imo, describe the **purpose** of a coinbase transation, not how it actually fulfills that purpose.  The same applies to other concepts as well.